library/init/meta/interactive.lean

```diff
@@ -1020,50 +1134,93 @@ structure unfold_config extends simp_config :=
 namespace interactive
 open interactive interactive.types expr
 
+/--
+Given defined constants ``e₁ ... eₙ``, ``unfold e₁ ... eₙ`` iteratively unfolds all occurrences in the target of the main goal, using equational lemmas associated with the definitions.
```

Comment:
Are the double backticks intended?

Reply:
Good catch. These are a holdover from the .rst version. I fixed those and a couple of others.